### ISSUE TITLE
Audio messages are not playing on Desktop 

### ISSUE THREAD

**Issue by simonricoo** [status-im/status-desktop#1947]
<img width="826" alt="Screenshot 2021-03-02 at 12 29 19" src="https://user-images.githubusercontent.com/52249767/109642225-f7ceec00-7b52-11eb-877c-5ed01103df6a.png">

**Comment by jrainville** [comment]
I've been investigating audio messages on Desktop and I've discovered that only Linux works.

Both Windows and Mac error with with " Failed to load media"/"Resource allocation error".

I have no idea what's the cause. I tried multiple types and other properties and nothing works.

All posts I see online using Audio use directly a sound file. For example, using an .mp3 works perfectly, but using straight data like we do doesn't.

I'm really not sure what to do. I know on Ubuntu I actually had to install an Encoder, so maybe that's what's missing also, but if so, it's really not legit to assume people will have that encoder installed

It could also be a bug with Qt itself
